apps/sso/components/otp-input/otp-input.spec.tsx

```diff
@@ -1,17 +1,17 @@
-import { render } from '@testing-library/react';
-
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
 import OTPInput from './otp-input';
 
-describe('LoginCard', () => {
-  it('should render successfully', () => {
-    const { baseElement } = render(
-      <OTPInput
-        setPinReady={undefined}
-        code={undefined}
-        setCode={undefined}
-        maxLength={undefined}
-      />
-    );
-    expect(baseElement).toBeTruthy();
-  });
-});
+const Story: ComponentMeta<typeof OTPInput> = {
+  component: OTPInput,
+  title: 'OTPInput',
+};
+export default Story;
+
+const Template: ComponentStory<typeof OTPInput> = (args) => (
+  <OTPInput {...args} />
+);
+
+export const Primary = Template.bind({});
+Primary.args = {
+  userTag: '',
+};
```

Comment:
this is a .spec.ts file which is for testing though; this seems like Storybook which should be in a .storybook.ts file instead